Makefile

```diff
@@ -20,7 +20,7 @@
 # Note that we also amend the path to include /usr/local/opt/ruby/bin, which
```

Comment:
Sorry. My comment wasn't left earlier for some reason. Just wonder if we should update this comment to mention the rbenv change.